static/templates/invitation_failed_error.handlebars

```diff
@@ -4,3 +4,8 @@
     <li>{{this}}</li>
     {{/each}}
 </ul>
+{{#if check_if_admin}}
+<p>{{admin_error}}<a href="#organization/deactivated-users-admin"> organization settings</a>.</p>
```

Comment:
This approach doesn't internationalize properly; instead, you should just generate the whole string inside `invite.js` using `i18n.t()`.